src/analyze/analyze.c

```diff
@@ -1737,6 +1751,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  calendar SPEC...         Validate repetitive calendar time events\n"
                "  service-watchdogs [BOOL] Get/set service watchdog state\n"
                "  timespan SPAN...         Validate a time span\n"
+               "  security [UNIT...]       Analyze security of unit\n"
```

Comment:
This should take a path like `verify` does. It is very natural to run this on a single file that hasn't yet been installed into the file search path.

Reply:
uh, but that'd be a major rework. The tool currently just reads the bus properties of PID 1. making it analyze unit files means doing the parsing client side. 

Both come with pros and cons. In the current implementation it's easy to check transient units. 

I see your point though, but I'd really leave that for a later PR, where the "struct security_info" could be initialized directly by the unit file parser?